bc_obps/registration/models.py

```diff
@@ -1,3 +1,135 @@
+    business_number = models.CharField(max_length=1000, db_comment="")
+    business_structure = models.CharField(max_length=1000, db_comment="") 
+    mailing_address = models.CharField(max_length=1000, db_comment="") 
+    bceid = models.CharField(max_length=1000, db_comment="") 
+    parent_operator = models.ForeignKey('self', on_delete=models.CASCADE, related_name='parent', db_comment="",blank=True, null=True)
+    relationship_with_parent_operator = models.CharField(max_length=1000, db_comment="",blank=True, null=True) 
+    compliance_obligee = models.ForeignKey('self', on_delete=models.CASCADE, related_name='obligee', db_comment="")
+    date_aso_became_responsible_for_operator = models.DateTimeField(db_comment="")
+    # status = models.CharField(max_length=50, choices=Statuses.choices, default=Statuses.PENDING, db_comment="")
+    documents  = models.ManyToManyField(Document,blank=True, null=True)
+    contacts  = models.ManyToManyField(Contact)
+    operators = models.ManyToManyField(User, through="UserOperator")
+    class Meta:
+        db_table_comment = "Operators (also called organizations)"
+        # don't need indexes if we end up using `unique`
+        indexes = [
+            models.Index(fields=["parent_operator"], name="parent_operator_idx"),
+            models.Index(fields=["compliance_obligee"], name="compliance_obligee_idx")]
+
+class UserOperator(models.Model):
+    
+    # class Roles(models.TextChoices):
+    #     ADMIN = 'admin', 'Admin'
+    #     SUPERADMIN = 'superadmin', 'SuperAdmin'
+    
+    class Statuses(models.TextChoices):
+        PENDING = 'pending', 'Pending'
+        APPROVED = 'approved', 'Approved'
+        REJECTED = 'rejected', 'Rejected'
+
+    users = models.ForeignKey(User, on_delete=models.CASCADE)
+    operators = models.ForeignKey(Operator, on_delete=models.CASCADE)
+    role = models.CharField(max_length=1000, db_comment="")
+    status = models.CharField(max_length=1000, choices=Statuses.choices, default=Statuses.PENDING, db_comment="")
+    user_is_aso = models.BooleanField
+    user_is_third_party = models.BooleanField
+    proof_of_authority = models.ForeignKey(Document, on_delete=models.DO_NOTHING, db_comment="")
+    class Meta:
+        db_table_comment = "Through table to connect Users and Operators and track access requests"
+        indexes = [
+            models.Index(fields=["users"], name="user_operator_user_id_idx"),
+            models.Index(fields=["operators"], name="user_operator_operator_id_idx")]
+    
+class Operation(models.Model):
+    operator_id = models.ForeignKey(Operator, on_delete=models.CASCADE, db_comment="")
+    name = models.CharField(max_length=1000, db_comment="")
+    operation_type = models.CharField(max_length=1000, db_comment="")
+    naics_code = models.ForeignKey(NaicsCode, on_delete=models.CASCADE, db_comment="")
+    first_year_of_commercial_operation = models.IntegerField(db_comment="")
```

Comment:
nit: could be `PositiveSmallIntegerField`

Reply:
SMEs decided we don't need this column anymore, but I'll keep in mind for the future, thanks!